src/desktop/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "trinity-desktop",
   "productName": "Trinity",
   "version": "0.5.1",
-  "private": true,
```

Comment:
Could you try reverting this change to see if it makes the Snyk test pass?

Reply:
Still fails :/